aptos-move/aptos-vm-types/src/storage.rs

```diff
@@ -178,10 +184,55 @@ impl StoragePricingV2 {
     }
 }
 
+// No storage curve. New gas parameter representation.
+#[derive(Debug, Clone)]
+pub struct StoragePricingV3 {
```

Comment:
is this something new added in this PR or moved from somewhere else?

Reply:
It's newly added. Although its logic should be identical to V2 at the latest gas feature version. I created V3 so it can return abstract usage. (Notice that V2 is still returning a concrete number. It's a bit tricky to unify the two code paths.)

Reply:
Is it possible to move this change to a separate PR? Seems an orthogonal change?